compiler/util/clangUtil.cpp

```diff
@@ -54,6 +54,8 @@
 
 #include "llvmDebug.h"
 
+#include "llvm/Support/TargetRegistry.h"
```

Comment:
I must be missing something - why is this #include now necessary? You didn't change anything else in this file.